python/fledge/services/core/api/support.py

```diff
@@ -191,9 +204,17 @@ async def get_syslog_entries(request):
             response['count'] = total_lines
             cmd = template.format(valid_source[source], _SYSLOG_FILE, total_lines - offset, limit)
         else:
-            cmd = non_total_template.format(valid_source[source], _SYSLOG_FILE, offset, limit)
-        a = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE).stdout.readlines()
+            scriptPath = os.path.join(_SCRIPTS_DIR, "common", "get_logs.sh")
+            # cmd = non_total_template.format(valid_source[source], _SYSLOG_FILE, offset, limit)
+            pattern = '({})\[.*\].*{}:'.format(valid_source[source], levels)
+            cmd = '{} -offset {} -limit {} -pattern \'{}\' -logfile {} -source {} -level {}'.format(scriptPath, offset, limit, pattern, _SYSLOG_FILE, source, level)
+            _logger.info('********* non_totals=true: new shell command: {}'.format(cmd))
```

Comment:
log should be with debug if really required

Reply:
yes, kept it info for testing round.